### ISSUE TITLE
Add case sensitivity support

### ISSUE THREAD

**Issue by fotonmoton** [vsch/laravel-translation-manager#123]
If two keys differ from each other only by case package treat them as the same. I think problem in mysql> like in this [issue](https://github.com/barryvdh/laravel-translation-manager/issues/165). Package version: 2.3.8, Laravel: v5.3.31

**Comment by vsch** [comment]
@fotonmoton, yes it makes sense. MySQL treats text as case insensitive with default collation on most databases.
